lib/pry/pry_class.rb

```diff
@@ -1,7 +1,17 @@
 require 'pry/config'
 class Pry
 
-  HOME_RC_FILE = ENV["PRYRC"] || "~/.pryrc"
+  HOME_RC_FILE = proc do
+    if ENV.has_key?('PRYRC')
+      ENV['PRYRC']
+    elsif File.exist?(File.expand_path('~/.pryrc'))
+      '~/.pryrc'
+    elsif ENV.has_key?('XDG_CONFIG_HOME') && ENV['XDG_CONFIG_HOME'] != ''
+      ENV['XDG_CONFIG_HOME'] + '/pry/config'
+    else
+      '~/.config/pry/config'
```

Comment:
👍 

Can we specify a link to the standard above this whole expression, by the way?